### ISSUE TITLE
Unable to get route key

### ISSUE THREAD

**Issue by si3nloong** [ItalyPaleAle/svelte-spa-router#132]
```
const routes = {
  "/": HomePage,
  "/example/:id": ExamplePage,
}

<Router {routes} on:routeLoaded={routeLoaded} />
```

Currently there have no way to get the route key `/example/:id` in event `on:routeLoaded`, i hope you expose the value in the `on:routeLoaded` event

**Comment by ItalyPaleAle** [comment]
The routeLoded callback receives an `event` parameter that contains a variety of values. In your case, you need to use `event.detail.location`. Docs and example here: https://github.com/ItalyPaleAle/svelte-spa-router/blob/master/Advanced%20Usage.md#routeloaded-event

**Comment by si3nloong** [comment]
This is ain't the value i expected, the value i want is the raw value `/example/:id` instead of `/example/123`

**Comment by ItalyPaleAle** [comment]
I see. In this case I suggest using the wrap function to attach custom data to a route. You’ll see the custom data in event.detail.customData. By setting a different value for each path, you can achieve the same result: https://github.com/ItalyPaleAle/svelte-spa-router/blob/master/Advanced%20Usage.md#route-pre-conditions

**Comment by si3nloong** [comment]
yeah, currently I using `wrap` function to overcome it, but I hope it include inside the event object (it did make sense we need to check on the route key, because that is a unique identifier for the specific route)

**Comment by ItalyPaleAle** [comment]
Ok it makes sense. I'll reopen this and look at what I can do

**Comment by ItalyPaleAle** [comment]
Ok I've implemented it in the v3.0 branch. I am hoping to have the time to finish work on that branch soon and make a release :)

**Comment by si3nloong** [comment]
👍
